GCDS.NetTemplate/Views/Shared/GCDS.NetTemplate/_Layout.Basic.cshtml

```diff
@@ -15,8 +15,10 @@
 	<title>@template.PageTitle</title>
 
 	@await Html.PartialAsync("ExtHeadSettings.cshtml", template.Settings)
-
-	@Html.Raw(template.HeadElements.Render())
+	@foreach (var headElement in template.HeadElements)
+	{
+		@await Html.PartialAsync("ExtHtmlElement.cshtml", headElement)
+	}
```

Comment:
I wonder if there is a good way to centralize common logic for the Template layouts.